code/modules/research/designs/bluespace_designs.dm

```diff
@@ -13,17 +13,17 @@
 
 /datum/design/bag_holding
 	name = "Bag of Holding"
-	desc = "A backpack that opens into a localized pocket of Blue Space."
+	desc = "A large bag that opens into a localized pocket of Blue Space."
 	id = "bag_holding"
-	req_tech = list("bluespace" = 8, "materials" = 5, "engineering" = 5, "plasmatech" = 6)
+	req_tech = list("bluespace" = 8, "materials" = 6, "engineering" = 7, "plasmatech" = 6)
 	build_type = PROTOLATHE
 	materials = list(MAT_GOLD = 3000, MAT_DIAMOND = 1500, MAT_URANIUM = 250, MAT_BLUESPACE = 2000)
 	build_path = /obj/item/storage/backpack/holding
 	category = list("Bluespace")
 
 /datum/design/bag_holding_large
-	name = "Bag of Holding (large things)"
-	desc = "A backpack that opens into a localized pocket of Blue Space. This variant can hold huge items, at the cost of capacity"
+	name = "Duffelbag of Holding"
```

Comment:
Surely these should have different sprites? I feel like they need different sprites.

Reply:
nevermind